### PR TITLE
Apply YAS formatting

### PR DESCRIPTION
**Pull request by omus**
Should be applying the YAS style guide before our first real release.

### CODE REVIEW

File: src/Ray.jl [hunk 1]
```diff
@@ -14,7 +14,7 @@ using Logging
 using LoggingExtras
 using Pkg
 using Serialization: Serialization, AbstractSerializer, Serializer, deserialize,
-    reset_state, serialize, serialize_type, ser_version, writeheader
+                     reset_state, serialize, serialize_type, ser_version, writeheader
 using Sockets: IPAddr, getipaddr
 
 import ray_julia_jll as ray_jll
```
Comment (ararslan) [inline src/Ray.jl:20]:
```suggestion
using ray_julia_jll: ray_julia_jll as ray_jll
```
Looks like JuliaFormatter doesn't recognize this case

Comment (omus) [reply src/Ray.jl:20]: TIL

**Comment by omus** [comment]
> I assume the formatter was applied here so I'll pre-approve to not block the release

Yes it was. I've also introduced a format check workflow to enforce it

**Comment by kleinschmidt** [comment]
> I've also introduced a format check workflow to enforce it

The auto-formatter action isn't very useful, it just tells you if it failed and which file, but doesn't suggest how to fix it.  It's also hard to run locally (you need to setup your own environmetn with formatter and run it in the same way that the action does.  Better IMO would be to use a separate `/format` project that has a `run.jl` script so you can run it locally in exactly the same way.  The action also doesn't specify what format to use.  I'll make a follow up PR to implement these suggestions!

**Comment by kleinschmidt** [comment]
Ah I see now, it's specified in the JuliaFormatter.toml file

**Pull request merged**